DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mdl
 Title: Modern model matrices
-Version: 0.0.0.9000
+Version: 0.0.1
```

Comment:
```suggestion
Version: 0.0.0.9001
```

In the R circles I run in, we tend to reserve 3-part Version numbers for CRAN releases, and increment a fourth component to indicate changes in dev version.